src/components/supporters/SupporterCard/index.js

```diff
@@ -30,12 +30,16 @@ module.exports = React.createClass({
           <Footprint
             data={ props.footprint }
             compact={ true }
-            size={ 120 }
+            size={ 140 }
             userImage={ props.image }
             userName={ props.name }
             userUrl={ props.url } />
 
-          <a href={ props.url } className="SupporterCard__userName">{ props.name }</a>
+          <a href={ props.url } className="SupporterCard__userDetails">
+            <div className="SupporterCard__name">{ props.name }</div>
+            { (props.charity && !props.hideCharity) && <div className="SupporterCard__charity">For <strong>{ props.charity }</strong></div> }
+            { (props.event && !props.hideEvent) && <div className="SupporterCard__event">Through <strong>{ props.event }</strong></div> }
```

Comment:
Can you please add `charity`, `event`, `hideCharity` and `hideEvent` to propTypes?